doc/man/scc-hypervisor-collector.1.md

```diff
@@ -0,0 +1,216 @@
+  collected details to the SUSE Customer Center.
+
+See **scc-hypervisor-collector(5)** for details about the possible
+configuration settings.
+
+## CONFIGURATION MANAGEMENT
+The configuration settings can be specified in a single YAML file,
+or as a set of YAML files whose content will be merged together in
+lexical sort order.
+
+The latter scheme allows the configuration settings to be split up
+into multiple files, e.g. credentials can be specified in one file,
+and the hypervisor backends in one or more files.
+
+When splitting the hypervisor backend details among multiple files,
+the **backends** lists in each file will be merged together to form
+a single combined list; exact duplicates will be ignored but partial
+duplicates will result in errors.
+
+## ACCESS AND OWNERSHIP
+For security reasons only the non-root user that is running the
+**scc-hypervisor-collector** command should be able to access the
+specified configuration files.
+
+## CONFIGURATION VALIDATION
+The **--check** option can be utilised to check if the specified
+configuration settings are valid, or will report any errors that
+it detected.
+
+## SUPPORTED HYPERVISORS
+
+The following hypervisor types are supported:
+
+* VMWare vCenter (type **VMware**)
+* Libvirt (type **Libvirt**)
+
+Each hypervisor type has specific configuration settings that must
+be provided to permit the relevant details to be retrieved; these
+settings are documented in **scc-hypervisor-collector(5)**.
+
+# EXIT CODES
+
+**scc-hypervisor-collector** sets the following exit codes:
+
+* 0:  Run completed successfully, or configuration settings
+      are valid if check mode (**--check**) was specified.
+* 1:  An error occurred.
+
+# IMPLEMENTATION
+
+**scc-hypervisor-collector(1)** is implemented in Python.
+It communicates with the SUSE Customer Center via a RESTful
+JSON API over HTTP using TLS encryption.
+
+# ENVIRONMENT
+
+**scc-hypervisor-collector(1)** respects the HTTP_PROXY environment
+variable.  See https://www.suse.com/support/kb/doc/?id=000017441 for
+more details on how to manually configure proxy usage.
+
+# FILES AND DIRECTORIES
+
+**~/.config/scc-hypervisor-collector/**
+: Default configuration directory containing YAML configuration files,
+  merged together in lexical sort order. Directory and files must be
+  owned by, and only accessible by, the user running the
+  **scc-hypervisor-collector(5)** command.
+
+**~/scc-hypervisor-collector.log**
+: Default log file which will be automatically rotated and compressed
+  if it gets too large. Log files must be owned by, and only accessible
+  by, the user running the **scc-hypervisor-collector(5)** command.
+  Will be created with appropriate permissions if no log file exists.
+
+**/var/lib/scchvc/.config/scc-hypervisor-collector/**
+: Configuration directory that the **scc-hypervisor-collector.service(8)**
+  checks for configuration settings. Directory and files must be owned
+  by, and only accessible by, the **scchvc** user.
```

Comment:
Should be consistent here. either all scchvc user or all  scchvc service account
I think user can be confusing since id can mean the person or account depending on context,
Is the scchvc a no-login account? If so, is config checking done via "sudo su  scchvc" Just trying to figure out how the --check process would work.

Reply:
Move to scc-hypervisor-collect.service.8.md